src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IArgument.cs

```diff
@@ -739,5 +778,714 @@ Element Values(1): IConversionExpression (ConversionKind.CSharp, Implicit) (Oper
+        {
+            var il = @"
+.class public auto ansi beforefieldinit P
+       extends [mscorlib]System.Object
+{
+  .custom instance void [mscorlib]System.Reflection.DefaultMemberAttribute::.ctor(string) 
+           = {string('Item')}
+
+  .method public hidebysig specialname rtspecialname 
+          instance void  .ctor() cil managed
+  {
+    // Code size       8 (0x8)
+    .maxstack  8
+    IL_0000:  ldarg.0
+    IL_0001:  call       instance void [mscorlib]System.Object::.ctor()
+    IL_0006:  nop
+    IL_0007:  ret
+  } // end of method P::.ctor
+
+  .method public hidebysig specialname instance int32 
+          get_Item([opt] int32 i,
+                   [opt] int32 j) cil managed
+  {
+    .param [1] = int32(0x00000001)
+    .param [2] = int32(0x00000002)
+    // Code size       7 (0x7)
+    .maxstack  1
+    .locals init ([0] int32 V_0)
+    IL_0000:  nop
+    IL_0001:  ldc.i4.0
+    IL_0002:  stloc.0
+    IL_0003:  br.s       IL_0005
+    IL_0005:  ldloc.0
+    IL_0006:  ret
+  } // end of method P::get_Item
+  
+  .method public hidebysig specialname instance void 
+          set_Item([opt] int32 i,
+                   [opt] int32 j,
+                   int32 'value') cil managed
+  {
+    .param [1] = int32(0x00000003)
+    .param [2] = int32(0x00000004)
+    // Code size       2 (0x2)
+    .maxstack  8
+    IL_0000:  nop
+    IL_0001:  ret
+  } // end of method P::set_Item
+
+
+  .property instance int32 Item(int32,
+                                int32)
+  {
+    .get instance int32 P::get_Item(int32,
+                                    int32)
+    .set instance void P::set_Item(int32,
+                                   int32,
+                                   int32)
+  } // end of property P::Item
+} // end of class P
+"; 
+
+            var csharp = @"
+class C
+{
+    public void M1()
+    {
+         P p = new P();
+         /*<bind>*/p[10]/*</bind>*/ = 9;
+    }
+}
+";
+            string expectedOperationTree = @"
+IIndexedPropertyReferenceExpression: System.Int32 P.this[[System.Int32 i = 3], [System.Int32 j = 4]] { get; set; } (OperationKind.IndexedPropertyReferenceExpression, Type: System.Int32) (Syntax: 'p[10]')
+  Instance Receiver: ILocalReferenceExpression: p (OperationKind.LocalReferenceExpression, Type: P) (Syntax: 'p')
+  Arguments(2): IArgument (ArgumentKind.Explicit, Matching Parameter: i) (OperationKind.Argument) (Syntax: '10')
+      ILiteralExpression (Text: 10) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 10) (Syntax: '10')
+    IArgument (ArgumentKind.DefaultValue, Matching Parameter: j) (OperationKind.Argument) (Syntax: 'p[10]')
+      ILiteralExpression (OperationKind.LiteralExpression, Type: System.Int32, Constant: 4) (Syntax: 'p[10]')";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTestWithIL<ElementAccessExpressionSyntax>(csharp, il, expectedOperationTree, expectedDiagnostics);
```

Comment:
>VerifyOperationTreeAndDiagnosticsForTestWithIL<ElementAccessExpressionSyntax>(csharp, il, expectedOperationTree, expectedDiagnostics); [](start = 12, length = 134)

Please adjust the test so that we can observe what value is going to be used at runtime. Either compare IL for M1, or adjust the code so that it prints the value when executed.